docs/conf/portland/2021/news/events-and-activities.rst

```diff
@@ -0,0 +1,65 @@
+This will give you the opportunity to set up your Hopin account and troubleshoot any access issues that you might have. 
+
+If you don't get an email from us by April 22nd, please email us and we'll help you out!
+
+Welcome Wagon guide and orientation
+-----------------------------------
+
+Each year we create a guide to help our attendees get the most out of the conference.
+Following the success of last year's conferences and based on your feedback, we expanded `our Welcome Wagon guide <https://www.writethedocs.org/conf/portland/2020/welcome-wagon/>`_ to help you get the most out of our event this year.
+
+We have a lot of information about how to use Hopin, and lots of tips and tricks for the best way to plan your days.
+
+This year we are also happy to announce that the Welcome Wagon team will hold a **live orientation session** for the virtual platform! The session will be held on Sunday during Writing Day, and will give you a chance to walk through the virtual event and ask all your questions. 
+
+Check out the `Schedule page <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for exact time and location.
+
+All our events and activities
+-----------------------------
+
+Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.
+
+* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required. Sign-up for lightning talks will be posted on Monday and Tuesday morning, and the selected talks will be announced before the lunch break on each day.. 
+* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
+* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.
+* **Social event**. On Monday night the main stage will be closed, and we will switch to social mode! We will be using a different platform for some of the social event, and we will post the link for the social space during the day on Monday. You're welcome to run sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.
```

Comment:
```suggestion
* **Social event**. On Monday night the main stage will be closed, and we will switch to social mode! We will be using a spatial chat platform for the social event this year, and we will post the link for the social space during the day on Monday. It will be much closer to a "real life" social where attendees can move between different rooms and group conversations, and the `Pac-Man Rule <https://www.ericholscher.com/blog/2017/aug/2/pacman-rule-conferences/>`_ even works!
```